### ISSUE TITLE
@apply rules regression

### ISSUE THREAD

**Issue by jakubpawlowicz** [jakubpawlowicz/clean-css#874]
via @web-padawan in https://github.com/jakubpawlowicz/clean-css/issues/872#issuecomment-275320567

3.4.24
```
.focused-line {
    @apply(--layout-fit);
    border-bottom: 2px solid var(--paper-input-container-focus-color, --primary-color);
    -webkit-transform-origin: center center;
    transform-origin: center center;
    -webkit-transform: scale3d(0, 1, 1);
    transform: scale3d(0, 1, 1);
    @apply(--paper-input-container-underline-focus)
}
.unfocused-line {
    @apply(--layout-fit);
    border-bottom: 1px solid var(--paper-input-container-color, --secondary-text-color);
    @apply(--paper-input-container-underline)
}
:host([disabled]) .unfocused-line {
    border-bottom: 1px dashed;
    border-color: var(--paper-input-container-color, --secondary-text-color);
    @apply(--paper-input-container-underline-disabled)
}
```

4.0.1
```
.focused-line {
    @apply(--layout-fit);
    border-bottom: 2px solid var(--paper-input-container-focus-color, --primary-color);
    -webkit-transform-origin: center center;
    transform-origin: center center;
    -webkit-transform: scale3d(0, 1, 1);
    transform: scale3d(0, 1, 1) @apply(--paper-input-container-underline-focus)
}
.unfocused-line {
    @apply(--layout-fit);
    border-bottom: 1px solid var(--paper-input-container-color, --secondary-text-color) @apply(--paper-input-container-underline)
}
:host([disabled]) .unfocused-line {
    border-bottom: 1px dashed;
    border-color: var(--paper-input-container-color, --secondary-text-color) @apply(--paper-input-container-underline-disabled)
}  
```

Looks like the case is when `@apply` comes after line ending with closing paren, which I can confirm for:
* CSS custom properties (`var`), 
* `transform` functions (`scale3d` etc) 
* color functions (`rgba`).

**Comment by jakubpawlowicz** [comment]
Fixed in 4.0.2.

**Comment by web-padawan** [comment]
Thanks! Will check it tomorrow.
